subprojects/test-runner/service/src/main/kotlin/com/avito/runner/service/worker/device/Device.kt

```diff
@@ -35,9 +36,16 @@ interface Device {
 
     /**
      * @return `to` path
+     *
+     * todo deprecate in favor of [pullDirRecursively]
      */
     fun pull(from: Path, to: Path): Result<File>
 
+    /**
+     * @return `hostDir` path
+     */
+    fun pullDirRecursively(deviceDir: Path, hostDir: Path, validator: PullValidator): Result<File>
```

Comment:
```suggestion
    fun pullDir(deviceDir: Path, hostDir: Path, validator: PullValidator): Result<File>
```
Copying dirs already implies recursion

Reply:
fixed